pyproject.toml

```diff
@@ -17,11 +17,13 @@ certifi = ">=2023.7.22"
 frozendict = "^2.3.2"
 pillow = "^9.0.0" # TODO: We may want to mark pillow (and numpy) as extra (https://python-poetry.org/docs/master/pyproject#extras)
 pydantic = "^1.7.4"
-python = ">=3.7.0,<4.0"
+python = ">=3.9.0,<4.0"
 python-dateutil = "^2.8.2"
 requests = "^2.28.2"
 urllib3 = "^1.26.9"
 typer = "^0.9.0"
+Sphinx = "7.2.6"
```

Comment:
These should be moved into "dev" dependencies - we don't need them in the SDK at runtime, which is what happens in this section.